pom.xml

```diff
@@ -470,6 +494,7 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-mapreduce</artifactId>
           </exclusion>
+          <!--
```

Comment:
I believe if not excluded here then they come in with the wrong scope.  If necessary they should be added as test scope.  They are runtime scoped in adam-assembly.

In any case, things still aren't right with logging.  There are often multiple slf4j warnings at runtime and log4j configuration doesn't seem to work for me.